code/network/multi_pxo.cpp

```diff
@@ -2560,29 +2471,23 @@ void multi_pxo_process_channels()
  */
 void multi_pxo_channel_refresh_servers()
 {
-	pxo_channel *lookup;
 	filter_game_list_struct filter;
 
-	// traverse the list of existing channels we know about and query the game tracker about them
-	lookup = Multi_pxo_channels;
-
-	if (lookup == NULL) {
+	if (Multi_pxo_channels.size() == 0) {
 		return;
 	}
 
-	do {
-		if ( strlen(lookup->name) ) {
+	// traverse the list of existing channels we know about and query the game tracker about them
+	for (auto channel : Multi_pxo_channels) {
```

Comment:
Should be `auto&` (or `auto &channel`). One less thing for Cyborg to deal with in the next Coverity scan.

Reply:
For sure. Done.